### ISSUE TITLE
[feature request] Switching to history tab opens the history search 

### ISSUE THREAD

**Issue by ueberBrot** [brave/brave-browser#5028]
It would be nice that when you open the brave history tab with CTRL-H or via the menu to automatically be in the search field so that you can type your search query. To do that now you have to manually click on the search icon in the right top corner of the history tab.

That is the standard in Chrome, Chromium and also Firefox. Would be nice to have it here as well.

**Comment by skylord123** [comment]
Yeah I use history a lot and having to touch my mouse just makes it that much slower. As a web designer this actually is pretty annoying.

**Comment by rebron** [comment]
cc: @bradleyrichter @rossmoody to revisit search

**Comment by wodim** [comment]
@skylord123 you don't have to touch your mouse, just press Ctrl+F.

Also: bump. This would be nice in the bookmark manager as well

**Comment by skylord123** [comment]
> @skylord123 you don't have to touch your mouse, just press Ctrl+F.
> 
> Also: bump. This would be nice in the bookmark manager as well

I actually figured this out hah. It is still annoying to have an extra step but at least it is quick.

Would definitely prefer it auto opening though (there isn't any reason not to that I can think of).

**Comment by bsclifton** [comment]
This has been implemented (or inherited from upstream)
